src/main/scala/edu/berkeley/cs/rise/opaque/benchmark/Benchmark.scala

```diff
@@ -46,12 +49,17 @@ object Benchmark {
     //   if (spark.sparkContext.isLocal) 1 else spark.sparkContext.defaultParallelism
 
     // Warmup
-    LogisticRegression.train(spark, Encrypted, 1000, 1)
-    LogisticRegression.train(spark, Encrypted, 1000, 1)
+//    LogisticRegression.train(spark, Encrypted, 1000, 1)
```

Comment:
Why did you comment this out?

Reply:
Commented it out previously while figuring if I can just run everything in benchmark.scala and never reverted. Reverted now.